### PR TITLE
SAK-30480 - ckeditor isn't present in resources (removing jquery from

### PR DESCRIPTION
**Pull request by jonespm**
launch)

### CODE REVIEW

File: reference/library/src/webapp/editor/ckeditor.launch.js [hunk 1]
```diff
@@ -30,6 +30,21 @@ function findPos(obj) {
         return [curleft,curtop];
     }
 }
+
+//http://stackoverflow.com/a/1038781/3708872
```
Comment (buckett) [inline reference/library/src/webapp/editor/ckeditor.launch.js]: Is it worth making this only available inside the launch function as some other javascript may have a getWidth() in the global scope that may cause problems?

**Comment by ottenhoff** [comment]
Okay LGTM

Why doesn't Resources have jquery?

**Comment by buckett** [comment]
ckeditor is loaded before jQuery is on the page and the init of the ckeditor is half way down the page.

**Comment by jonespm** [comment]
Yeah, that's fine, I moved both the findPos and getWidth into the launch. 

Yeah, there are a few pages where jquery either isn't available or is loaded after for whatever reason. It's just not guaranteed to be available.

**Pull request merged**